NuRadioMC/utilities/Veff.py

```diff
@@ -468,38 +222,48 @@ def get_Veff_single(filename, trigger_names, trigger_names_dict, trigger_combina
         fin.attrs['phimin']
```

Comment:
It looks like this should be phimin = fin.attrs['phimin']
Same in line 224

Reply:
yes, thanks